### PR TITLE
Update install defn; add activations

### PR DESCRIPTION
**Pull request by fbertsch**
Depends on https://github.com/mozilla/bigquery-etl/pull/3579

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [x] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [x] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [x] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [x] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql [hunk 1]
```diff
@@ -60,15 +62,18 @@ SELECT
   stats.impressions AS ad_impressions,
   install_dou_metrics.new_profiles_sum AS installs,
   install_dou_metrics.dau_sum AS dous,
+  install_dou_metrics.dau_sum AS dau,
```
Comment (chelseatroy) [inline sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql:65]: Do we mean to have the same value from `install_dou_metrics.dau_sum` in two separate columns by different names here?

File: sql/moz-fx-data-shared-prod/fenix/marketing_attributable_metrics/view.sql [hunk 2]
```diff
@@ -6,5 +6,7 @@ SELECT
   1 AS dau,
   IF(is_new_profile, 1, 0) AS new_profiles,
   searches AS search_count,
+  IF(is_new_install, 1, 0) AS new_installs,
```
Comment (chelseatroy) [inline sql/moz-fx-data-shared-prod/fenix/marketing_attributable_metrics/view.sql:9]: This is purely for my edification; what distinguishes a _new_ install? Is this, like, a person who has never installed before and is _not_ reinstalling to update?

File: sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql [hunk 3]
```diff
@@ -60,15 +62,18 @@ SELECT
...
   install_dou_metrics.revenue_generating_ad_clicks_sum AS revenue_generating_ad_clicks,
+  install_dou_metrics.new_installs_sum,
+  install_dou_metrics.activations_sum,
   -- Cost per-install for each campaign ($/new profiles)
   CASE
     WHEN install_dou_metrics.new_profiles_sum = 0
       THEN 0
     ELSE stats.cost_micros / install_dou_metrics.new_profiles_sum
-  END AS cost_per_install_micros,
+  END AS cost_per_new_profile_micros,
```
Comment (chelseatroy) [inline sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql:76]: Ah, renamed here so we can add the metric on line 98 by the original name, gotcha

**Review by chelseatroy — COMMENTED** [review]
The one question about the dupe column info, but can approve once we've ironed that out~

File: sql/moz-fx-data-shared-prod/fenix/marketing_attributable_metrics/view.sql (hunk 2 above)
Comment (fbertsch) [reply sql/moz-fx-data-shared-prod/fenix/marketing_attributable_metrics/view.sql:9]: I just added this, but this is basically to account for the [missing Ad clicks & CDOU incident](https://docs.google.com/document/d/1r1RcRstytYN21sxl82Cbr4ueGErLe6Q7ho_hsQxrg2o/edit#). We have a pretty good sense of when an ID is regenerated vs. a new install, so I'm counting _just_ new installs here (or approximating them, at least).

**Review by chelseatroy — APPROVED** [review]

**Pull request merged**